plugins/node/opentelemetry-instrumentation-redis-4/README.md

```diff
@@ -71,6 +71,20 @@ const redisInstrumentation = new RedisInstrumentation({
 });
 ```
 
+## Semantic Conventions
+
+This package uses `@opentelemetry/semantic-conventions` version `1.22+`, which implements Semantic Convention [Version 1.7.0](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.7.0/semantic_conventions/README.md)
+
+Attributes collected:
+
+| Attribute              | Short Description                                                            |
+| ---------------------- | ---------------------------------------------------------------------------- |
+| `db.statement`         | The database statement being executed.                                       |
+| `db.system`            | An identifier for the database management system (DBMS) product being used.  |
+| `db.connection_string` | The connection string used to connect to the database (without credentials). |
```

Comment:
nit: can you move this up to keep this list ordered alphabetically?

Reply:
done!